src/templates/coffee.njk

```diff
@@ -1,7 +1,8 @@
 {% extends '_base.njk' %}
 
 {% block title %}Coffee{% endblock %}
-
+{% block navCoffee %}active{% endblock %}
+{% block navCoffee2 %}link-light{% endblock %}
 {% block content %}
```

Comment:
Mám tendenci mezi bloky dávat mezery (prázdné řádky) pro přehlednost, ale ok, ničemu to nevadí.